### ISSUE TITLE
Issue when the type contains "implements" instruction

### ISSUE THREAD

**Issue by Tibiritabara** [erebus1/graphene-federation#6]
The issue is the next one: if your type contains an implements, like the ones generated by graphene-django, the library _sdl generation appends the @key("id") suffix before the implements, when it should be present afterwards.

This is caused by the service.py, line 29
```
for entity_name, entity in custom_entities.items():
        type_def = "type %s" % entity_name
        repl_str = "%s %s " % (type_def, entity._sdl)
        pattern = re.compile(type_def)
        string_schema = pattern.sub(repl_str, string_schema)
```

As we see on the type_def var, it does not consider the fact that there might be an "implement" or other keywords after the type name. 

The solution is quite simple with a regex

**Comment by erebus1** [comment]
Thanks for reporting!
https://github.com/erebus1/graphene-federation/pull/7

Seem there is the same issue with @external
I'll check it a bit later

**Comment by Tibiritabara** [comment]
I saw the fix, but it was also hitting auto-generated types by graphene. I just added a space on the regular expression. It is available on: #8

**Comment by erebus1** [comment]
> Seems there is the same issue with `@extend`
> I'll check it a bit later

fixed: https://github.com/erebus1/graphene-federation/pull/16
